engine/classes/ElggEntity.php

```diff
@@ -1686,7 +1686,14 @@ protected function update() {
 		unset($persisted_entity);
 
 		if ($allow_edit) {
-			$allow_edit = _elgg_services()->events->trigger('update', $this->type, $this);
+			$allow_edit = elgg_trigger_before_event('update', $this->type, $this);
```

Comment:
@beck24 I think leaving "update" as is and adding "update:after" is an OK compromise. We'd need to mention it in the RST docs